tests/modules/test_JMeterExecutor.py

```diff
@@ -588,6 +588,15 @@ def test_distributed_th_hostnames(self):
         for writer in writers:
             self.assertEqual('true', writer.find('objProp/value/hostname').text)
 
+    def test_distributed_props(self):
+        self.obj.execution.merge({"scenario": {"script": __dir__() + "/../jmeter/jmx/http.jmx"}})
+        self.obj.distributed_servers = ["127.0.0.1", "127.0.0.1"]
+        self.obj.settings['properties'] = BetterDict()
+        self.obj.settings['properties'].merge({"a": 1})
+
+        self.obj.prepare()
+        self.obj.startup()
+
```

Comment:
All works, but there is one problem. This test passed without any new logic in jmeter.py. Yes, it covers if-clause, but what does it check exactly?